File: squares-of-a-sorted-array/squares-of-a-sorted-array.java
class Solution {
    public int[] sortedSquares(int[] nums) {
        
        List<Integer> ls=new ArrayList<Integer>();
        for(int i=0;i<nums.length;i++)
            ls.add(nums[i]*nums[i]);
        Collections.sort(ls);
        int i=0;
        for(int x:ls)
            nums[i++]=x;
        return nums;
    }
}

File: squares-of-a-sorted-array/squares-of-a-sorted-array.cpp
class Solution {
public:
    vector<int> sortedSquares(vector<int>& A) {
        int size=A.size();
        vector<int> v;
        vector<int>::iterator it;
        it=v.begin();
        for(int i=0;i<size;i++)
        {
           int j=(A.at(i)*A.at(i));
            v.push_back(j);
        }
        sort(v.begin(),v.end());
        return v;
        // for(int i=0;i<size;i++)
        // {
        //     cout<<A.at(i)<<" ";
        // }
    }
};